src/operators/mod.rs

```diff
@@ -57,6 +66,7 @@ pub(crate) fn reduce(
 
         if should_emit_and_discard_aggregator {
             let emit = (key.clone_ref(py), updated_aggregator).to_object(py).into();
+            debug!("{}, reducer:{:?}, key:{:?}, emit:{:?}, complete:{:?}", log_func!(), reducer, key, emit, should_emit_and_discard_aggregator);
```

Comment:
This might better reflect the logic here if it logged the call to `is_complete` and its arguments? Right now I can't tell what function is being called nor what its arguments are, which is the meat of the second step of the reduce operator.

Reply:
You mean what `is_complete` function is being called?